### ISSUE TITLE
[Feature Request] Add a "download album" button.

### ISSUE THREAD

**Issue by Sacha-Ringenbach** [koel/player#63]
Right now, there is no way to download an entire album at once, you have to download each song manually.

**Comment by spyfly** [comment]
Would be a welcome addition

**Comment by Kinzi** [comment]
Upvote ;)

**Comment by phanan** [comment]
@CCO19 That's not true. When you start the app, if there's no connection, the "You are offline" screen will appear, allowing you to view and play downloaded content.

<img width="301" height="auto" alt="Image" src="https://github.com/user-attachments/assets/28747d1b-aa36-4a9d-a14b-571169dbd527" />

**Comment by CCO19** [comment]
@phanan, thanks for your feedback. That’s good to know! So the app detects when the smartphone’s network connection is down. When the server is down, I always get the authentication screen. But next time I’ll try putting my smartphone in airplane mode to see if that helps.

**Comment by phanan** [comment]
@CCO19 I've been using this while flying since forever :)

**Comment by CCO19** [comment]
> [@CCO19](https://github.com/CCO19) I've been using this while flying since forever :)

It's works! It’s a bit of a hassle, though, because Plane Mode cuts off the Android Auto connection, and it’s a pain to reconnect while driving 😅 So you have to remember to do it before you get behind the wheel so you don’t waste too much time.

In my opinion, an “Offline Mode” button would be more convenient.

**Comment by phanan** [comment]
You can go to Library -> Downloaded.
